src/org/omegat/gui/common/EntryInfoSearchThread.java

```diff
@@ -110,10 +113,11 @@ public void run() {
         SwingUtilities.invokeLater(new Runnable() {
             public void run() {
                 if (isEntryChanged()) {
+                    pane.abortSearch(OStrings.getString("ENTRYINFOSEARCH_ENTRY_CHANGED"));
                     return;
                 }
                 if (ferror != null) {
-                    pane.setError(ferror);
+                    pane.abortSearch(ferror.getLocalizedMessage());
```

Comment:
```suggestion
                    pane.setError(ferror);
```

change is redundant. It should be reverted.